ko/documentation/installation/index.md

```diff
@@ -282,3 +289,7 @@ $ sudo make install
 [opensolaris-pkg]: http://opensolaris.org/os/project/pkg/
 [gentoo-ruby]: http://www.gentoo.org/proj/en/prog_lang/ruby/
 [homebrew]: http://brew.sh/
+[terminal]: : https://ko.wikipedia.org/wiki/%EB%8B%A8%EB%A7%90_%EC%97%90%EB%AE%AC%EB%A0%88%EC%9D%B4%ED%84%B0_%EB%AA%A9%EB%A1%9D
+[download]: /ko/downloads/
+[installers]: /en/documentation/installation/#installers
```

Comment:
We can link to `/ko/documentation/intallation/#installers`.

Reply:
8ac1c199ff092e2 🙈